packages/valory/skills/decision_maker_abci/redeem_info.py

```diff
@@ -124,14 +125,17 @@ def __hash__(self) -> int:
         """Custom hashing operator."""
         return hash(self.fpmm.condition.id) + hash(self.fpmm.question.id)
 
-    @property
-    def claimable_amount(self) -> int:
-        """Get the claimable amount of the current market."""
-        return int(self.outcomeTokenMarginalPrice * self.outcomeTokensTraded)
-
     @property
     def is_winning(self) -> bool:
         """Return whether the current position is winning."""
         our_answer = self.outcomeIndex
         correct_answer = self.fpmm.current_answer_index
         return our_answer == correct_answer
+
+    @property
+    def claimable_amount(self) -> int:
+        """Get the claimable amount of the current market."""
+        amount = int(self.outcomeTokenMarginalPrice * self.outcomeTokensTraded)
```

Comment:
Is this correct? I am considering "outcomeTokensTraded" in this script https://github.com/valory-xyz/trader-quickstart/blob/04b185e0db4d65449bc86f89b194842eb765b78b/trades.py#L441-L442

Reply:
I believe, you need to multiply the price by the number of conditional tokens to get the result.

Reply:
I agree with @Adamantios here

Reply:
Updated in #105.